readme.md

```diff
@@ -116,13 +116,13 @@ The user selects an extension from this list, and the communication starts from
 async function startExtensionSession(
     extension: InjectedWindowProvider,
     dAppName: string,
-    dAppEncryptionKeyId: string, 
+    dAppEncryptionKeyUri: string, 
     challenge: string
 ): Promise<PubSubSession> {
     try {
-        const session = await extension.startSession(dAppName, dAppEncryptionKeyId, challenge);
+        const session = await extension.startSession(dAppName, dAppEncryptionKeyUri, challenge);
         
-        // Resolve the `session.encryptionKeyId` and use this key and the nonce 
+        // Resolve the `session.encryptionKeyUri` and use this key and the nonce 
```

Comment:
This breaking change will go into the next version of the specification, please revert it here